### DISCUSSION TITLE
ReferenceError: im__vite_ssr_import_meta__ is not defined for import.meta.env.VITE_

### DISCUSSION THREAD

**Discussion by YajanaRao** [vitest-dev/vitest#5442] [Q&A]
I am using a environment variable in vue file which is working fine with vite. When I run vitest I get the following error
`ReferenceError: im__vite_ssr_import_meta__ is not defined`

where as following is working in vitest:
expect(import.meta.env.VITE_APP_).toBe('')

**Comment by sheremet-va** [comment]
Do you have a similar problem? https://github.com/vitest-dev/vitest/issues/5387

**Reply by YajanaRao** [reply]
I ended up mocking env variables
